src/modules/database/generators/BaseGenerator.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include "faker-cxx/Database.h"
+#include "faker-cxx/Helper.h"
+#include "faker-cxx/Number.h"
+#include "faker-cxx/types/Country.h"
+#include "faker-cxx/types/Sex.h"
+#include "../data/ColumnData.h"
+#include "../src/common/FormatHelper.h"
+
+#include <string>
+#include <vector>
+
+namespace faker
+{
+class BaseGenerator
+{
+protected:
+    SqlCommandType CommandType;
+    faker::Country country;
+    faker::Sex sex;
```

Comment:
why do you keep sex and country in base generator? there may be situation when we take like `orders` table and there would not be any of that used

Reply:
Yeah, this brings a good point. I original reason why I had the country and sex in the base generator is because I wanted to generate the data consistently for all columns. I also reuse the same column collection for all tables. Perhaps, I should use a separate collection (subset of the original collection) for each table.

Reply:
Ok nice